### DISCUSSION TITLE
Welcome to MediaCrawler Discussions!

### DISCUSSION THREAD

**Discussion by NanmiCoder** [NanmiCoder/MediaCrawler#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MorningYin** [comment]
这玩意怎么停止啊

**Reply by Alluresmith** [reply]
手动打断就行，ctrl+c

**Reply by NanmiCoder** [reply]
ctrl + c

**Comment by AnsonStudying** [comment]
抖音中搜索中文关键字好像并不行

**Reply by NanmiCoder** [reply]
这个是由于抖音那边更新了接口，最新代码已经修复了

**Comment by RICKDONG123** [comment]
(venv) D:\MediaCrawler>python main.py
  File "D:\MediaCrawler\main.py", line 1
    --ks --lt qrcode --type search
IndentationError: unexpected indent  最后一步啦， 标准格式是什么 ，爬 快手的关键词

**Comment by new-pj** [comment]
可以获取用户的小红薯帖子对应用户的粉丝量吗

**Comment by nhg-GitHu** [comment]
能爬取抖音视频下面的全部评论吗？
我自己写了一个 skill，只能爬取部分评论
